java/src/jmri/implementation/configurexml/MatrixSignalMastXml.java

```diff
@@ -30,7 +30,7 @@ public MatrixSignalMastXml() {
     public Element store(Object o) { // from mast p to XML
         MatrixSignalMast p = (MatrixSignalMast) o;
 
-        Element e = new Element("signalmast");
+        Element e = new Element("matrixsignalmast");
```

Comment:
As I mentioned earlier, I don't think that we need to create a new versioned schema for this change, rather update the existing one. That the version number of the current schema is several years old does not mean that this is frozen from that point, rather at that version a breaking change was introduced.
Looking at http://jmri.org/help/en/html/doc/Technical/XmlSchema.shtml specifically the discussion around when to version the schema, would tend to favour updating the existing wherever possible, rather than creating a new one.